addons/l10n_es/migrations/13.0.4.0/openupgrade_analysis.txt

```diff
@@ -58,24 +58,8 @@ DEL account.account.tag: l10n_es.mod_303_36_37
 DEL account.account.tag: l10n_es.mod_303_38_39
 DEL account.account.tag: l10n_es.mod_303_40_41
 NEW account.account.template: l10n_es.account_common_4301
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_10_inv
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_10b
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_10s
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_21_inv
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_21b
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_21s
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_4_inv
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_4b
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_4s
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_10b
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_10s
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_21b
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_21isp
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_21s
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_4b
-DEL account.fiscal.position.tax.template: l10n_es.fptt_not_subject_tai_ventas_4s
-DEL account.fiscal.position.template: l10n_es.fp_not_subject_tai
-DEL account.tax.template: l10n_es.account_tax_template_p_iva0_ns_b
+NEW account.fiscal.position.template: l10n_es.fp_intra_private
+NEW account.tax.template: l10n_es.account_tax_template_s_iva0_ns
```

Comment:
I think this is not real, but outdated 12.0 branch

Reply:
I'm pretty sure this one is due to an outdated 12.0 branch

Reply:
In openupgrade v12, there aren't commits https://github.com/odoo/odoo/commit/624cd93f07d40e6b6d1ecaaa5810505eb1c8a6d4 and https://github.com/odoo/odoo/commit/207207d9dc1c2e0c18c6f3b48078314ced8b38cb.

Reply:
Here you have the same record for v12:

https://github.com/odoo/odoo/blob/12.0/addons/l10n_es/data/account_tax_data.xml#L786

Reply:
Yeah, but the commit https://github.com/odoo/odoo/commit/624cd93f07d40e6b6d1ecaaa5810505eb1c8a6d4 was added in odoo v12 after the las update upstream of openupgrade v12.

Reply:
So this is why I'm saying that 12.0 branch (of OpenUpgrade) is outdated... When you are migrating from 12.0 to 13.0, you are not using the outdated OpenUpgrade 12.0 branch for the previous DB version, but Odoo/OCB one.

Reply:
Yeah, one thing is when you are migrating. But oher thing is to do an analysis. And the analysis is done between openupgrades, always. 😅
And yesterday I didn't do a update upstream for v12 because I did it 20 days ago, which I thought it wouldn't be necessary to update again so recently.

Reply:
But you have removed only one line. At least the other NEW line is also incorrect. Please update 12.0 OpenUpgrade branch or check correctly the changes.

Reply:
Wait, I searched in odoo 12.0 and I didn't find that fp_intra_private template. So I thought it's really new in v13, thus I didn't delete that line. I am wrong?

Reply:
OK, you are right. I checked bad.